docs/docs/dev_docs/cli/updating.md

```diff
@@ -4,44 +4,24 @@ title: Updating
 
 ## Quick Reference
 
-- Aztec Sandbox
+Inside your project run:
 
 ```shell
-  /bin/bash -c "$(curl -fsSL 'https://sandbox.aztec.network')"
+cd your/aztec/project
+npx @aztec/cli@latest update . --contract src/contract1 --contract src/contract2
```

Comment:
@alexghr the sandbox has to running for this command to work, correct?

Reply:
There's two possibilities:
1. if the project has `@aztec/aztec-sandbox` listed as a dependency in package.json then it will compare against that version
2. otherwise it grabs the version of the running sandbox. If sandbox isn't running it exits and prints the error message:
```
Could not connect to Sandbox running on <url>
```